setup.py

```diff
@@ -15,6 +15,7 @@
     'html5lib<2',
     'lxml<5',
     'six<2',
+    'furl>=2.0.0',
```

Comment:
While the lower constraint is definitely needed, I have been placing upper bounds on dependencies.  It's not very common, but the basic idea is that a major version increment is backwards incompatible.  Since no guarantees can be made about whether current code will be compatible with a backwards incompatible change, I find it prudent to place the upper bound.

As an example, the transition from furl 1.x to 2.x made all parameters keyword parameters.  If this library had been using 1.x, calls to furl would have likely thrown exceptions when version 2.0 was released.  An upper bound prevents that from happening.  The downside to this is that this library must stay current with its dependencies (check out of date dependencies with "pip list -o").  A good unit test suite with complete coverage will help with problem and this library has that.